hardware/plugins/PluginProtocols.cpp

```diff
@@ -282,28 +282,28 @@ namespace Plugins {
 		}
 	}
 
-#define ADD_BYTES_TO_DICT(pDict, key, value) \
-		{	\
-			PyObject*	pObj = Py_BuildValue("y#", value.c_str(), value.length());	\
-			if (PyDict_SetItemString(pDict, key, pObj) == -1)	\
-				_log.Log(LOG_ERROR, "(%s) failed to add key '%s', value '%s' to dictionary.", __func__, key, value.c_str());	\
-			Py_DECREF(pObj); \
+	static void AddBytesToDict(PyObject* pDict, const char* key, const std::string& value)
```

Comment:
Could you undo these changes as they are not about the logging ? If needed, please create a separate pull request

Reply:
Probably all changes in this class